src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibleWidget.ts

```diff
@@ -87,11 +89,22 @@ export abstract class TerminalAccessibleWidget extends DisposableStore {
 		this._element.replaceChildren(this._editorContainer);
 		this._xtermElement.insertAdjacentElement('beforebegin', this._element);
 
-		if (this._focusContextKey) {
+		if (this._focusContextKey && this._focusLastLineContextKey) {
```

Comment:
It looks like `_focusContextKey` is always truth so I don't think this if needs to be here? It was a little confusing seeing an if like this in the ctor.

This also means that `_focusedContextKey` doesn't need the `?`